Channels/Operation/01_ManageOffers.md

```diff
@@ -0,0 +1,339 @@
+
+4. Click the *Select connection* drop-down list and select the connection for which you want to create the offer. All available connections are displayed in the list.   
+    The *Select attribute set* drop-down list is unlocked.
+
+5. Click the *Select attribute set* drop-down list and select the appropriate attribute set for the offer. All available attribute sets for the selected connection are displayed.
+
+6. Click the [SAVE] button.   
+    The *Create offer* view is displayed. The *Attributes* tab is preselected.
+
+    ![Create offer](../../Assets/Screenshots/Channels/Offers/Offers/CreateOffer.png "[Create offer]")
+
+    > [Info] The fields displayed in the *Attributes* tab differ depending on the selected connection.
+
+7. Enter the product data in the fields of the *Attributes* tab. You must at least enter a product name, an EAN code, a price, a tax class and a tax rate.
+
+8. Click the [SAVE] button in the upper right corner.   
+    The offer has been created. The *Create offer* view is closed.
+
+    > [Info] By default, the initial offer status of the new offer is **Not available**. You have to activate the offer to sell it on a marketplace, see [Change the offer status](#change-the-offer-status).
+
+
+
+## Edit an offer
+
+After you have created an offer, you can edit it. However, not all attributes are editable. If an attribute can be edited in an offer or not, depend on several points, for instance, on the corresponding connection settings or if the attribute is mapped from a PIM product.   
+Note that any changes to an offer will trigger an automatic upload, so that any changes are applied to the products in the marketplace.
+In general, it is recommended to maintain or change offers that are mapped from a PIM product in the *PIM* module.
+
+#### Prerequisites
+
+At least one offer has been created, see [Create an offer](#create-an-offer).
+
+#### Procedure
+
+*Omni-Channel > Offers > Tab OFFERS*
+
+![Offers](../../Assets/Screenshots/Channels/Offers/Offers/Offers.png "[Offers]")
+
+1. Click the offer you want to edit in the list of offers.   
+    The *Edit offer* view is displayed. The *Attributes* tab is preselected.
+
+    > [Info] The tabs in the *Edit offer* view as well as the fields displayed in the *Attributes* tab differ depending on the selected connection.
+
+    ![Edit offer](../../Assets/Screenshots/Channels/Offers/Offers/EditOffer.png "[Edit offer]")
+
+2. Edit the desired data of the product in the corresponding fields and tabs.
+
+3. Click the [SAVE] button in the upper right corner.   
+    The changes have been saved. The *Edit offer* view is closed. The automatic upload has been triggered.
+
+    > [Info] Switch to *Offer Upload > Tab SCHEDULED UPLOADS* to check the queue for all scheduled offer uploads, see [Check the offer upload](./03_CheckOfferUpload.md).
```

Comment:
if the queue is empty prior to editing an offer, the queue may already be empty again after the client has saved an offer and refreshed that grid since the upload may have already gone through. Maybe worth a note so that people don't wonder why their changed offers never show up there.